### LidarArray.msg

Header header
Lidar[] lidars